source/partials/release_notes/_release-17-9-0.md.erb

```diff
@@ -0,0 +1,60 @@
+<h4>Improvements</h4>
+* <%= link_to_commit '928c5856c2bc73aa92c1a8eb1a86d8094a9fe806', 'Group elastic profile by plugin-id ([#3772](https://github.com/gocd/gocd/issues/3772))'%>
```

Comment:
Please see the older release notes as to how to link to issues.

Add screenshots for user facing improvements with a line or two of text?